src/initializeUploadcare.js

```diff
@@ -0,0 +1,21 @@
+import {getScript} from './getScript'
+
+function loadWidget(options) {
+  const widgetSrc = `https://ucarecdn.com/libs/widget/${WIDGET_VERSION}/uploadcare.full.min.js`
```

Comment:
Founded =) 

Why are you use `<script>` instead of npm packages?

Reply:
@Zmoki because of `strict mode`. By default NetlifyCMS starters with Hugo and Gatsby built with `strict mode` enabled. I think it's weird to force users to disable `strict mode` for using our plugin.